packages/core/src/Account.ts

```diff
@@ -363,7 +364,8 @@ export class Account extends TypedEventEmitter<Events> {
     await this.apiClient.transport.http.associateClientWithSession(client.id);
 
     await this.service.proteus.initClient(this.storeEngine, this.apiClient.context);
-    if (this.service.mls && mlsConfig) {
+
+    if ((await this.isMLSActiveForClient()) && this.service.mls && mlsConfig) {
```

Comment:
this is already `this.service.mls` checked in isMLSActiveForClient

Reply:
Double check doesn't hurt to satisfy TS. The function is public and available for web app, so we need to check the service availability there as well.